.config/forest.dic

```diff
@@ -94,3 +94,4 @@ VRF
 WebAssembly
 WebSocket
 zstd
+Sha256
```

Comment:
Perhaps in the docs let's use the longer form, i.e., `SHA-256`. Alternatively, it may be too much hassle and you could just use backticks in the docs themselves.